### ISSUE TITLE
Nodester road maps.

### ISSUE THREAD

**Issue by FLYBYME** [nodester/nodester#413]
What does the next 6 months look like for the development of nodester. I have noticed not much works has been done on the source code. Is Nodester ever going to reach version 2?

**Comment by alejandro** [comment]
I don't think we are going to reach version 2 since nodester is part now of AppFog (http://blog.nodester.com/post/30434216604/nodester-joins-appfog). AppFog uses a different architecture so they probably will put all they efforts in that architecture.

**Comment by FLYBYME** [comment]
Yeah this is why I asked. I dont see appfog going anywhere with Nodester. Has any work been done on version two? The readme says some stuff about the next version, but I can't find any code for it.
